scripts/ci/build-installers.sh

```diff
@@ -48,8 +48,8 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$TARGET_ARCH" \
     -s "$(pwd)" \
-    -c 'make installers-all'
+    -c 'make electron-develop installers-all'
```

Comment:
And the solution is https://github.com/resin-io/etcher/pull/1514. We can manually remove the entries without ever running npm.